src/test/regress/expected/multi_metadata_sync.out

```diff
@@ -59,10 +59,11 @@ ALTER ROLE CURRENT_USER WITH PASSWORD 'dummypassword';
 -- Show that, with no MX tables, activate node snapshot contains only the delete commands,
 -- pg_dist_node entries, pg_dist_object entries and roles.
 SELECT unnest(activate_node_snapshot()) order by 1;
-                                                                                                                                                                             unnest
+                                                                                                                                                                                                                                                       unnest
```

Comment:
From those changes I expect to have a synced colocation group info on a newly added node, though having an explicit test checking the consistency between newly added nodes' `pg_dist_colocation` and coordinator nodes' `pg_dist_colocation` would be helpful.

Reply:
adjusted the test at:
https://github.com/citusdata/citus/pull/5742/files#diff-a9c19a48e54fdd8f889ce75c4ebc7d037d397a40fb69432ffdcb28d613d21d91R518